onadata/libs/utils/logger_tools.py

```diff
@@ -468,6 +490,8 @@ def safe_create_instance(username, xml_file, media_files, uuid, request):
     except InstanceEmptyError:
         error = OpenRosaResponseBadRequest(
             _(u"Received empty submission. No instance was created"))
+    except InstanceEncryptionError as e:
+        error = OpenRosaResponseBadRequest(e)
```

Comment:
Would you also not need to apply `text(e)` as is the pattern in this section, see the trend below where the error variable `e` is concerned.